docs/reference/ml/anomaly-detection/apis/get-datafeed-stats.asciidoc

```diff
@@ -103,6 +103,24 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=node-transport-address]
 ====
 --
 
+`running_state`::
+(object) An object containing the running state for this {dfeed}. It is only
+provided if the {dfeed} is started.
++
+--
+[%collapsible%open]
+====
+`is_real_time`:::
+(boolean) Indicates if the {dfeed} is "real-time"; meaning that the {dfeed}
+has no configured `end` time.
+
+`finished_look_back`:::
```

Comment:
Our existing docs don't use the term "lookback".  I am not sure we should introduce it now.

One solution would be to rename `is_real_time` to `real_time_configured` and `finished_look_back` to `real_time_running`.  Then that doesn't involve introducing a new public term.

Reply:
@droberts195 

While its true that we don't use it in docs, `lookback` is used in audit messages:

```
    public static final String JOB_AUDIT_DATAFEED_LOOKBACK_COMPLETED = "Datafeed lookback completed";
    public static final String JOB_AUDIT_DATAFEED_LOOKBACK_NO_DATA = "Datafeed lookback retrieved no data";
```

Reply:
I am happy to change to both to indicate `real_time`. But we do use lookback in user facing things.

Reply:
> `lookback` is used in audit messages

Oh, interesting.  I think we made the decision in 2016 not to use it anywhere user-facing, but those audit messages were added in 2017, by which time the 2016 decision had been forgotten.  I guess the lesson is that we should change our terminology in internal code as well as what's immediately user-facing to stop internal terminology leaking out later on.

I still think there is a benefit in not propagating the term to fields that will appear in every high level client's public API.  At the moment we could change the wording of those audit messages without making a breaking change, but the REST responses are harder to change once published.

Reply:
okey dokey, I will rename the variables. 

> I guess the lesson is that we should change our terminology in internal code as well as what's immediately user-facing to stop internal terminology leaking out later on.

100%. Renaming things everywhere prevents this sort of thing.